tests/features/tmgmt/tmgmt_dgt_connector.feature

```diff
@@ -17,6 +17,7 @@ Feature: TMGMT Poetry features
       | fr        |
     And I am logged in as a user with the "administrator" role
 
+  @javascript
```

Comment:
Since it is passing the Jenkins tests, including the ones for ec_europa, I suppose we are already handling this the right way. @voidtek did some work on this in previous ticket [NEPT-873, see last commits](https://github.com/ec-europa/platform-dev/pull/1475/commits).